test/scriptFlywheel/unit/liquidateTests.spec.ts

```diff
@@ -743,53 +645,62 @@ contract("SavingAccount.liquidate", async (accounts) => {
                     await savingAccount.deposit(addressDAI, ONE_DAI, { from: user1 });
-                    await savingAccount.deposit(ETH_ADDRESS, ONE_ETH, {
-                        from: user2,
-                        value: ONE_ETH,
-                    });
+                    await savingAccount.deposit(addressUSDC, ONE_USDC, { from: user2 });
+                    await savingAccount.deposit(addressDAI, ONE_DAI, { from: owner });
                     // 2. Start borrowing.
-                    const result = await tokenInfoRegistry.getTokenInfoFromAddress(addressDAI);
-                    const daiTokenIndex = result[0];
+                    const limitAmount = ONE_USDC.mul(await tokenInfoRegistry.priceFromIndex(1))
+                        .mul(new BN(60))
+                        .div(new BN(100))
+                        .div(await tokenInfoRegistry.priceFromIndex(0));
+                    const result = await tokenInfoRegistry.getTokenInfoFromAddress(addressUSDC);
+                    const usdcTokenIndex = result[0];
                     await accountsContract.methods["setCollateral(uint8,bool)"](
-                        daiTokenIndex,
+                        usdcTokenIndex,
                         true,
-                        { from: user1 }
-                    );
-                    await savingAccount.borrow(ETH_ADDRESS, borrowAmt, { from: user1 });
-
-                    await expectRevert(
-                        savingAccount.liquidate(user1, ETH_ADDRESS, addressDAI),
-                        "The borrower is not liquidatable."
+                        { from: user2 }
                     );
+                    await savingAccount.borrow(addressDAI, limitAmount, { from: user2 });
+                    // 3. Change the price.
+                    let updatedPrice = new BN(1);
+                    const originPrice = await mockChainlinkAggregatorforUSDC.latestAnswer();
+                    await mockChainlinkAggregatorforUSDC.updateAnswer(updatedPrice);
+                    // Check that borrow LTV > 95%
+                    const userBorrowVal = await accountsContract.getBorrowETH(user2);
+                    const getDeposits = await accountsContract.getDepositETH(user2);
+                    const userBorrowPower = await accountsContract.getBorrowPower(user2);
+                    let UB = new BN(userBorrowVal).mul(new BN(100));
+                    let UD = new BN(getDeposits).mul(new BN(95));
+                    console.log("userBorrowVal", userBorrowVal.toString());
+                    console.log("userBorrowPower", userBorrowPower.toString());
+                    console.log("UD", UD.toString());
+                    console.log("UB", UB.toString());
+                    expect(UB).to.be.bignumber.greaterThan(UD);
+                    await savingAccount.liquidate(user2, addressDAI, addressUSDC);
+                    await mockChainlinkAggregatorforUSDC.updateAnswer(originPrice);
```

Comment:
here also `updateAnswer` is used.

Reply:
Yes, have to remove those. Even I just saw while working on liquidate